projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -395,6 +405,8 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
   }
 
   public setDisabledState(shouldDisable: boolean | undefined): void {
+    this.disabledBeforeInit = !!shouldDisable;
```

Comment:
move this down into the next block so that it's not confusing for debugging

Reply:
I tried. Test is failing and our use case at cloudnc wasn't fixed when I moved it there

Reply:
Discussed IRL. I kept the same logic but renamed the variable to make more sense